detekt-rules-empty/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyKotlinFileSpec.kt

```diff
@@ -0,0 +1,36 @@
+package io.gitlab.arturbosch.detekt.rules.empty
+
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.test.assertThat
+import io.gitlab.arturbosch.detekt.test.compileAndLint
+import org.junit.jupiter.api.Test
+
+class EmptyKotlinFileSpec {
+    private val subject = EmptyKotlinFile(Config.empty)
+
+    private val codeWithPackageStatement = """
+            package my.package
+    """.trimIndent()
+
+    private val codeWithFunction = """
+            package my.package
+            
+            fun myFunction() {}
+    """.trimIndent()
+
+    @Test
+    fun `reports empty if file is blank`() {
+        val code = ""
+        assertThat(subject.compileAndLint(code)).hasSize(1)
```

Comment:
I'm not 100% sure why an empty file doesn't compile but that's not an issue for this PR. It's safe to just run the lint without compiling it.

```suggestion
        assertThat(subject.lint(code)).hasSize(1)
```